### DISCUSSION TITLE
Changed username

### DISCUSSION THREAD

**Discussion by Mimicthis** [community/community#23357] [New to GitHub]
<p>Hi!</p>
<p>I’ve changed my username. I only use github to report issues and ask questions in some forums.</p>
<p>Does it affect that?</p>

**Comment by canuckjacq** [comment]
<p>It shouldn’t affect any of that. Any @ mentions of your old username will no longer link through, but otherwise, the consequences of username changing are mostly to do with public links and repositories.</p>

**Comment by Mimicthis** [comment]
<p>Do you maybe know how to change the username on this forum? :slight_smile:</p>

**Comment by canuckjacq** [comment]
<p>Yes! You ask. Would you like me to make your usernames match?</p>

**Comment by Mimicthis** [comment]
<p>Yes, please! I would like you to change it!</p>

**Comment by canuckjacq** [comment]
<p>I was wrong :see_no_evil:</p>
<p>Fortunately I’ve found another way. You need to log out of the community, and then go to your GitHub app authorisation settings <a href="https://github.com/settings/apps/authorizations">here</a>.</p>
<p>Select “authorised GitHub apps” and then revoke access to the community. When you come back, you’ll need to re-authorise the community, and your username will be updated. :tada:</p>

**Comment by Mimicthis** [comment]
<p>Weeee, thank you so much. Done!!</p>

**Comment by TehBrian** [comment]
<p>Hi there,</p>
<p>I recently changed my GitHub username from “ItsTehBrian” to “TehBrian”, however my username on GitHub Community (this very site) is still “ItsTehBrian”. This issue becomes apparent when clicking on the “GitHub Profile” button on my Community profile, as the button redirects you to an invalid GitHub page.</p>
<p>Is there any way that my Community username could be changed to “TehBrian”?</p>
<p>Thanks!</p>

**Comment by tuves** [comment, marked as answer]
<p>Hi <span class="mention">@itstehbrian</span>,</p>
<p>Can you try following these steps and see if it works?</p>
<aside class="quote" data-post="1" data-topic="132014">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/changed-username/132014">Changed username</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    Hi! 
I’ve changed my username. I only use github to report issues and ask questions in some forums. 
Does it affect that?
  </blockquote>
</aside>

**Comment by TehBrian** [comment]
<p>Looks like that solved my issue perfectly, thank you <a class="mention" href="/tuves">@tuves</a>!</p>
